assets/js/Router.jsx

```diff
@@ -336,7 +336,7 @@ const Router = (props) => {
                                 <Profile user={user} {...props} />
                               )}
                             />
-                            {false && (
+                            {true && (
```

Comment:
swap back lool

Reply:
i cry

Reply:
i literally deployed true to staging too sigh